### PR TITLE
N°6869 - Update chinese translations for ProfilesMenu

### PR DESCRIPTION
**Pull request by chileeb**
In iTop 3.1.0, the menu named "ProfilesMenu" was originally translated as "首选项", but it is suggested that "角色" (Roles) would be a more suitable translation.
tks!

### CODE REVIEW

**Comment by Molkobain** [comment]
Hello @chileeb thanks for the PR!

I'll poke our other chinese translators to ensure that everyone is ok with this :)
Poke @bdejin @annProg @purplegrape 

Take care,
Guillaume

**Comment by annProg** [comment]
lgtm

In 3.0, `ProfilesMenu` was translated as '角色'. I think It's better translated as "角色"

ref https://github.com/Combodo/iTop/commit/68e1f359d32eb29a30dfed3e38361f852ecc8ad8#diff-a33c27cce85d81f1a69f9c47428f289dfab0bf899738a3a44ca0a921e2a77cb4

**Review by annProg — APPROVED** [review]

**Comment by chileeb** [comment]
> Hello @chileeb thanks for the PR!
> 
> I'll poke our other chinese translators to ensure that everyone is ok with this :) Poke @bdejin @annProg @purplegrape
> 
> Take care, Guillaume

hi，i want to know how to join chinese translators as a member?

**Comment by Molkobain** [comment]
> hi，i want to know how to join chinese translators as a member?

I don't know, besides mentioning them like I did on the previous post. That being said, annProg validated it, so we can say it's good enough that 2 people aggree :)

File: dictionaries/zh_cn.dictionary.itop.ui.php [hunk 1]
```diff
@@ -1737,9 +1737,9 @@
 	'Menu:UniversalSearchMenu+' => '搜索所有...',
 	'Menu:UserManagementMenu' => '用户管理',
 	'Menu:UserManagementMenu+' => 'Who can connect to '.ITOP_APPLICATION_SHORT.' and what can they do?~~',
-	'Menu:ProfilesMenu' => '首选项',
-	'Menu:ProfilesMenu+' => 'View existing Profiles~~',
-	'Menu:ProfilesMenu:Title' => '首选项',
+	'Menu:ProfilesMenu' => '角色',
+	'Menu:ProfilesMenu+' => '查看已有的角色~~',
```
Comment (Molkobain) [inline dictionaries/zh_cn.dictionary.itop.ui.php]:
```suggestion
	'Menu:ProfilesMenu+' => '查看已有的角色',
```

**Review by Molkobain — APPROVED** [review]

**Comment by Molkobain** [comment]
Damn it, I choose the wrong option by mistake! I should have done a "squash & merge".
Sorry team for the git log pollution 😢

**Comment by Molkobain** [comment]
@chileeb could you send me your fullname and postal address at guillaume.lajarige combodo.com so we can send you [contributors stickers](https://github.com/Combodo/iTop/blob/develop/CONTRIBUTING.md#-we-are-thankful)? :)

**Comment by bdejin** [comment]
> lgtm
> 
> In 3.0, `ProfilesMenu` was translated as '角色'. I think It's better translated as "角色"
> 
> ref [68e1f35#diff-a33c27cce85d81f1a69f9c47428f289dfab0bf899738a3a44ca0a921e2a77cb4](https://github.com/Combodo/iTop/commit/68e1f359d32eb29a30dfed3e38361f852ecc8ad8#diff-a33c27cce85d81f1a69f9c47428f289dfab0bf899738a3a44ca0a921e2a77cb4)

Agree. It was good as "角色". Should keep as it.

**Comment by bdejin** [comment]
> > Hello @chileeb thanks for the PR!
> > I'll poke our other chinese translators to ensure that everyone is ok with this :) Poke @bdejin @annProg @purplegrape
> > Take care, Guillaume
> 
> hi，i want to know how to join chinese translators as a member?

@chileeb itophub has a detailed instruction [https://www.itophub.io/wiki/page?id=3_1_0:customization:translation](url). At the very bottom you'll find the section "How to contribute to improve an existing language". Cheers.

**Comment by bdejin** [comment]
> > hi，i want to know how to join chinese translators as a member?
> 
> I don't know, besides mentioning them like I did on the previous post. That being said, annProg validated it, so we can say it's good enough that 2 people aggree :)

@Molkobain @annProg I saw some missing translations (probably new in 3.1?) as well as a few backward ones. Might worth a overall checking.

**Comment by Molkobain** [comment]
@bdejin Feel free to make a PR for them, you can find all new translations by searching for `~~` at the end of the translations, it means that they have not been translated yet.

**Pull request merged**